cabal.project

```diff
@@ -31,6 +31,7 @@ packages:
   , libs/wire-message-proto-lens/
   , libs/zauth/
   , services/brig/
+  , services/backend-notification-pusher/
```

Comment:
can we still rename this to `worker`, or something similarly general?  [discussion here](https://wearezeta.atlassian.net/wiki/pages/diffpagesbyversion.action?pageId=384928204&selectedPageVersions=71&selectedPageVersions=72)